src/catalog-debug/src/main.rs

```diff
@@ -359,6 +366,11 @@ async fn dump(
             retraction_count,
             entries,
         };
+
+        if consolidate {
+            assert_eq!(retraction_count, 0);
```

Comment:
Should this be `neq`?

Reply:
I don't think so. If all entries are consolidated then there should be no retractions since all retractions should exactly match some previous addition. Otherwise we'd end up with an invalid retraction.

Reply:
Maybe I should turn this into an `error!` though so that if there's an invalid retraction we can see it.

Reply:
Done